extensions/amp-story-shopping/0.1/amp-story-shopping-attachment.js

```diff
@@ -14,6 +19,58 @@ export class AmpStoryShoppingAttachment extends AmpStoryPageAttachment {
 
     /** @private {?Element} */
     this.container_ = null;
+
+    /** @private @const {?../../amp-story/1.0/amp-story-store-service.AmpStoryStoreService} */
+    this.storeService_ = null;
+
+    /** @private {?../../../src/service/localization.LocalizationService} */
+    this.localizationService_ = null;
+  }
+
+  /**
+   * Updates the CTA based on the shopping data.
+   * @param {string} pageIndex
+   * @param {string} pageId
+   * @private
+   */
+  updateCta_(pageIndex, pageId) {
```

Comment:
nit:
What does it update about it? 
Please be as descripte and concise as possible.
 
Adding one word to the function name saves reviewers/maintainers a mental cycle. 
updateCtaText